### ISSUE TITLE
Support pre release

### ISSUE THREAD

**Issue by rewop** [conventional-changelog/standard-version#105]
I have been using standard-version now for a few weeks, and I realised that there is no support for pre-releases such as `1.0.0-alpha-1`, `2.0.0-beta-1` (AFAIK).

Is there any plan to support them?

**Comment by stevemao** [comment]
duplicate of #83 ?

**Comment by rewop** [comment]
That is exactly what I meant. Didn't search the PRs apparently. To me this can be closed once the PR is merged.

**Comment by stevemao** [comment]
Let's move the discussion there :)
